pyproject.toml

```diff
@@ -28,7 +28,7 @@ include = ["lua", "resources", "deployment"]
 pandas = ">=2.2.3,<3.0.0"
 boto3 = "^1.26.163"
 botocore = "^1.29.163"
-protobuf = "^3.1"
+protobuf = "^4.25.8"
```

Comment:
@tkilias , @ahsimb we should do a major release for. note that this also affects the notebook-connector which currently has sagemake-extension as a dependency.